lib/scrum/priority_name.rb

```diff
@@ -0,0 +1,15 @@
+module Scrum
+  class PriorityName
+    PRIORITY_REGEX      = /^(?:\([\d.]+\) )?P(\d+): /
```

Comment:
I think the dot "." shouldn't be there.

The existing regex:
```

2.3.0 :025 > "(.) P3: ".match(/^(?:\([\d.]+\) )?P(\d+): /)
 => #<MatchData "(.) P3: " 1:"3"> 
```

without the dot:

```
2.3.0 :027 > "(.) P3: ".match(/^(?:\(\d+\) )?P(\d+): /)
 => nil 
2.3.0 :028 > "(3) P3: ".match(/^(?:\(\d+\) )?P(\d+): /)
 => #<MatchData "(3) P3: " 1:"3">
```

I don't think we want to catch this case: "(.) P3:"